docs/output.md

```diff
@@ -213,15 +218,39 @@ By default, nf-core taxprofiler will only provide the `.bam` file if host remova
 
 > ℹ️ minimap2 is not yet supported as a module in MultiQC and therefore there is no dedicated section in the MultiQC HTML. Rather, alignment statistics to host genome is reported via samtools stats module in MultiQC report.
 
-### Samtools stats
+### SAMtools view
+
+[SAMtools view](http://www.htslib.org/doc/samtools-view.html) views and comverts a `.sam`, `.bam`, or `.cram` alignment file.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `samtoolsstats`
+  - `<sample_id>.bam`: Alignment file in BAM format
+
+</details>
+
+### SAMtools bam2fq
+
+[SAMtools bam2fq](http://www.htslib.org/doc/1.1/samtools.html) converts a `.sam`, `.bam`, or `.cram` alignment file to FASTQ format
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `samtoolsstats`
+  - `<sample_id>.fq.zg`: Alignment file in FASTQ gzip format.
+
+</details>
+
+### SAMtools stats
```

Comment:
This will be able to be collapsed into one :)

```
### SAMtools fastq

[SAMtools fastq](http://www.htslib.org/doc/1.1/samtools.html) converts a `.sam`, `.bam`, or `.cram` alignment file to FASTQ format

<details markdown="1">
<summary>Output files</summary>

- `samtoolsstats`
  - `<sample_id>.fq.gz`: Alignment file in FASTQ gzip format.

</details>

This directory will be present and contain the unmapped reads from the FASTQ format from long-read minimap2 host removal (for short-read unmapped reads, see [bowtie2](#bowtie2), if `--save_hostremoval_unmapped`  is supplied.

```